docs/make.bat

```diff
@@ -1,38 +1,38 @@
-@ECHO OFF

```

Comment:
Every time I commit something, Git replaces CRLF with LF in this file. This is due to the `core.autocrlf` setting, which IMO is better to accept and live with? But I don't know if it makes a syntax difference in a .bat file.

```
warning: CRLF will be replaced by LF in docs/make.bat.
The file will have its original line endings in your working directory.
```